modules/gapi/src/api/gkernel.cpp

```diff
@@ -98,3 +98,8 @@ std::vector<cv::gapi::GBackend> cv::gapi::GKernelPackage::backends() const
 
     return std::vector<cv::gapi::GBackend>(unique_set.begin(), unique_set.end());
 }
+
+cv::GCompileArgs cv::compile_args(cv::gapi::GKernelPackage pkg)
+{
+    return cv::GCompileArgs{ cv::GCompileArg(std::move(pkg)) };
+}
```

Comment:
Just for note - there is a way not to add it to main library - you just can mark in header as `#if 0` and move implementation to `pyopencv_gapi.hpp`

Reply:
Cool! Fixed